MANUAL.txt

```diff
@@ -1783,6 +1783,10 @@ Pandoc uses these variables when [creating a PDF] with ConTeXt.
     from [ConTeXt ICC Profiles].  See also [ConTeXt PDFA] for more
     details.
 
+`title-meta`
+:   sets a title for the document. Can be used to skip generating a titlepage
+    while still setting a document name.
+
```

Comment:
Probably a bit too cryptic.  Maybe

> sets the title in the PDF metadata.  Normally this is automatically generated from `title`.  However, it may be set manually when `title` is omitted.